com.unity.render-pipelines.high-definition/Runtime/Sky/SkyManager.cs

```diff
@@ -913,7 +913,7 @@ public void RenderSky(HDCamera hdCamera, Light sunLight, RTHandle colorBuffer, R
 
         public void RenderOpaqueAtmosphericScattering(CommandBuffer cmd, HDCamera hdCamera,
                                                       RTHandle colorBuffer,
-                                                      RTHandle volumetricLighting,
+                                                      RenderTexture volumetricLighting,
```

Comment:
Why this change here? Why not a RTHandle  ?

Reply:
is this ok with dynamic resolution ?

Reply:
Yes, it should work. Forgot to rename.